src/main/src/main/java/org/geoserver/security/file/LockFile.java

```diff
@@ -91,20 +99,19 @@ public void writeLock() throws IOException{
         
         if (hasWriteLock()) return; // already locked
                         
-        if (lockFile.exists()) {             
-            LOGGER.warning("Cannot obtain  lock: " + lockFile.getCanonicalPath());
+        if (Resources.exists(lockFile)) {             
+            LOGGER.warning("Cannot obtain  lock: " + lockFile.path());
             Properties props = new Properties();
 
-            try (FileInputStream in = new FileInputStream(lockFile)) {
+            try (InputStream in = lockFile.in()) {
                 props.load(in);
             }
 
             throw new IOException(Util.convertPropsToString(props,"Already locked" ));
         } else { // success             
             writeLockFileContent(lockFile);
-            lockFileLastModified =lockFile.lastModified();
-            lockFile.deleteOnExit(); // remove on shutdown
-            LOGGER.info("Successful lock: " + lockFile.getCanonicalPath());
+            lockFileLastModified =lockFile.lastmodified();
```

Comment:
Missing space. Also, File.lastModified has only second precision under Java 7 on Linux so locking causes horrible race conditions. Fixing this is out of scope.